posthog/api/cohort.py

```diff
@@ -283,13 +286,45 @@ def persons(self, request: Request, **kwargs) -> Response:
         elif not filter.limit:
             filter = filter.shallow_clone({LIMIT: 100})
 
-        query, params = PersonQuery(filter, team.pk, cohort=cohort).get_query(paginate=True)
+        if posthoganalytics.feature_enabled(
+            "load-person-fields-from-clickhouse",
+            None if request.user.is_anonymous else request.user.distinct_id,
```

Comment:
The `None` here will raise an assertion error and fail the request, is this the intent?

Maybe worth bringing this outside the feature flag check, or get rid of it if users can't be anonymous here, since I expect this api requires auth.